docs/2_3_fmu_distribution.adoc

```diff
@@ -81,7 +84,8 @@ The FMU must be distributed with [underline]#at least# one implementation,
 in other words, either [underline]#sources# or one of the [underline]#binaries# for a particular machine.
 It is also possible to provide the sources and binaries for different target machines together in one ZIP file.
 The following names are standardized: for Windows: "win32",
-"win64"- for Linux: "linux32", "linux64", for Macintosh: "darwin32", "darwin64".
+"win64"- for Linux: "linux32", "linux64", for Macintosh: "darwin32", "darwin64" where "32" stands for "32-bit x86" and "64" for "64-bit x86" respectively.
+For any other platforms the "platform tuples" defined in the FMI 3.0 Specification, 2.5.1.4.1. Platform Tuple Definition, should be used.
```

Comment:
Maybe

```suggestion
For any other platforms the "platform tuples" defined in the FMI 3.0 Specification, 2.5.1.4.1. Platform Tuple Definition, or any updated "platform tuples" published by MAP FMI on fmi-standard.org, should be used.
```

Or words to that effect, since we suggested that additional tuple definitions might be given in the future without updating the FMI 3.0 spec...